packages/studio-base/src/panels/Plot/index.stories.tsx

```diff
@@ -1670,3 +1670,44 @@ export const PreloadedDataAndItsAbsoluteValue: StoryObj = {
     await ctx.parameters.storyReady;
   },
 };
+
+export const DifferentLineSizes: StoryObj = {
+  render: function Story() {
+    const readySignal = useReadySignal({ count: 3 });
+    const pauseFrame = useCallback(() => readySignal, [readySignal]);
+
+    return (
+      <PlotWrapper
+        pauseFrame={pauseFrame}
+        config={{
+          ...exampleConfig,
+          paths: [
+            {
+              value: "/some_topic/location.pose.velocity",
+              enabled: true,
+              showLine: false,
+              timestampMethod: "receiveTime",
+              lineSize: 1.4,
```

Comment:
Making the difference more obvious
```suggestion
              timestampMethod: "receiveTime",
              lineSize: 2.5,
```